tox.ini

```diff
@@ -65,8 +74,10 @@ commands =
 changedir = docs
 description = Invoke sphinx-build to build the HTML docs
 extras = dev
+deps = astropy!=4.0.1
 commands =
-    sphinx-build . ./_build/html -W -b html -d .docs/_build/.doctrees {posargs}
+    sphinx-build --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
+    python -c 'import pathlib; print("Documentation available under file://\{0\}".format(pathlib.Path(r"{toxinidir}") / "docs" / "_build" / "index.html"))'
```

Comment:
f-string?

Reply:
I tried, i couldn't work out the right escape sequence to get it to work.

Reply:
Oh I see ok.